theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/CurveEditorPopover/EasingOption.tsx

```diff
@@ -0,0 +1,59 @@
+import BasicTooltip from '@theatre/studio/uiComponents/Popover/BasicTooltip'
+import useTooltip from '@theatre/studio/uiComponents/Popover/useTooltip'
+import React from 'react'
+import styled from 'styled-components'
+import {handlesFromCssCubicBezierArgs} from './shared'
+import SVGCurveSegment from './SVGCurveSegment'
+import mergeRefs from 'react-merge-refs'
+import {COLOR_BASE, COLOR_FOCUS_OUTLINE} from './colors'
+
+const Wrapper = styled.div`
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+  aspect-ratio: 1;
+
+  background: ${COLOR_BASE};
+  border-radius: 2px;
+  cursor: pointer;
+
+  // The candidate preset is going to be applied when enter is pressed
+
+  &:focus {
+    outline: none;
+    border: 1px solid ${COLOR_FOCUS_OUTLINE};
+  }
+
+  &:hover {
+    background: rgba(255, 255, 255, 0.2);
+  }
+`
+
+type IProps = {
+  easing: {
+    label: string
+    value: string
+  }
+} & Parameters<typeof Wrapper>[0]
+
+const EasingOption: React.FC<IProps> = React.forwardRef((props, ref) => {
```

Comment:
A few visual suggestions:

* Hide the handles from each `EasingOption`?
* A more minimal design for the tooltip.
* Less padding for each `EasingOption`

<img width="651" alt="image" src="https://user-images.githubusercontent.com/593118/165189034-98b136c1-00b9-4bda-8858-54e3dbf6ce50.png">